Form-Controls/index.html

```diff
@@ -9,19 +9,74 @@
   <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+  <!-- <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+  <script src="index.js"></script> -->
   <header>
     <h1>Product Pick</h1>
   </header>
   <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
+    <form name="choosing" method="post" class="field-wrap">
+      <div class="formbox">
+      <div>
+          <label for="name">Name</label>
+          <input type="text" id="name" name="name" required> 
```

Comment:
What are the validation criteria for names in the requirements?

Reply:
just force to write name but if we want we can write a Pattern for the name
pattern="^[A-Za-z]+(([',. -][A-Za-z ])?[A-Za-z]*)*$"